content/api/commands/intercept.md

```diff
@@ -29,20 +29,30 @@ All intercepts are automatically cleared before every test.
 cy.intercept(url)
 cy.intercept(method, url)
 cy.intercept(routeMatcher)
+```
+
+See arguments [url](/api/commands/intercept#url-String-Glob-RegExp), [method](/api/commands/intercept#method-String) and [routeMatcher](/api/commands/intercept#routeMatcher-RouteMatcher)
```

Comment:
The `url` link doesn't work, it looks like there's a missing blank line after `</Alert>` on line 95.

Reply:
OK try now

Reply:
Awesome